challenge.py

```diff
@@ -0,0 +1,117 @@
+import os
+import re
+import numpy as np
```

Comment:
Using numpy to parse diffs seems a bit overkill, especially when using only their arrays.

How faster are we compared to plain python Lists? Is there a noticeable improvement to justify the dependency?

Reply:
I agree with you, in the beginning I was thinking to do it in other manner that's why I used numpy. So, now I change it to simple list.